components/mediation/data-publishers/org.wso2.micro.integrator.prometheus.publisher/src/main/java/org/wso2/micro/integrator/prometheus/publisher/service/MetricResource.java

```diff
@@ -85,4 +99,18 @@ public boolean invoke(MessageContext synCtx) {
         return true;
     }
 
+    private Set<String> parseQuery(String query) throws IOException {
+
+        Set<String> names = new HashSet<>();
```

Comment:
Shall we add a comment describing what we are doing here? What query parameters we are expecting?

Reply:
Fixed